src/Nethermind/Nethermind.Specs/RopstenSpecProvider.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Nethermind.Specs
 {
-    public class RopstenSpecProvider : ISpecProvider
+    public class MainnetSpecProvider : ISpecProvider
```

Comment:
@Coollaitar I didn't want to rename RopstenSpecProvider to MainnetSpecProvider. I wanted to use this class: https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Specs/MainNetSpecProvider.cs

Reply:
And sorry for the late response